ion/binaryreader_test.go

```diff
@@ -16,13 +16,171 @@
 package ion
 
 import (
+	"bytes"
 	"fmt"
 	"math"
 	"math/big"
 	"testing"
 	"time"
 )
 
+const binaryFilePath = "testdata/binary/"
+
+func TestReadBinarySingleSymbol(t *testing.T) {
+	fileBytes := loadFile(t, binaryFilePath+"single_symbol.bindat")
```

Comment:
`gofmt` has some [Opinions(tm)](https://github.com/golang/go/issues/12720) about this that have never made much sense to me, but are very solidly established.

(Any particular reason for loading this test data from a separate file instead of inlining it like the other tests here?)

Reply:
Removed test because it is covered by [TestBinaryReadSymbols](https://github.com/amzn/ion-go/blob/35232d755b0e08ebc28448694beec4801889c483/ion/binaryreader_test.go#L421)